gatk-launch

```diff
@@ -153,20 +165,39 @@ def getSparkSubmitCommand(sparkSubmitCommand):
     else:
         return sparkSubmitCommand
 
-def getLocalGatkRunCommand():
+def getLocalGatkRunCommand(javaOptions):
     localJarFromEnv = getJarFromEnv(GATK_LOCAL_JAR_ENV_VARIABLE)
     if localJarFromEnv is not None:
+
+        # Add java options to our packaged local jar options
+        if javaOptions is not None:
+            PACKAGED_LOCAL_JAR_OPTIONS.extend(javaOptions.split(' '))
```

Comment:
You should call `javaOptions.split()` instead of `javaOptions.split(' ')` in order to allow for the possibility of multiple whitespace characters between tokens in the user's string.